.github/ISSUE_TEMPLATE/user-story.md

```diff
@@ -0,0 +1,52 @@
+---
+name: User Story
+about: Propose an enhancement via a user story
+title: ''
+labels: kind/user-story
+assignees: ''
+
+---
+## User Story
+
+*Using the form below, relate the desired product behavior.  Feel free to
+additional, clarifying details below it if needed.  For more details on user
+stories, see this [article](https://www.atlassian.com/agile/project-management/user-stories).*
+
+**As a** <persona name>
+**I want** <desired behavior>
+**So that** <value of behavior> 
+
+### Test Scenarios
+
+*If desired, add one or more test scenarios required for user story acceptance.
+Ideally, they are executed by automated tests. For more details on Gerkin syntax,
+see this [article](https://cucumber.io/docs/gherkin/reference/)*
+
+**Given** <this setup/context>
+**When** <an action/event occurs>
+**Then** <the expected behavior/outcome>
+
+### Definition of Done
```

Comment:
I think we should change this for (or add) something called Acceptance Criteria. We should have a global Definition of Done for all stories (I think Maciek is working on one) that includes things that we have to do all the time such as add test automation, changelog entries, etc. I can definitely see where having a DoD checklist in the story could be helpful, but we're mixing up the definitions of DoD and Acceptance Criteria. Although the Acceptance Criteria could be covered by the testing scenario section as long as the PO or PM are writing the testing scenarios because they should be defining the acceptance criteria. See [this blog post](https://www.altexsoft.com/blog/business/acceptance-criteria-purposes-formats-and-best-practices/).

Reply:
I agree.